### ISSUE TITLE
World End Syndrome

### ISSUE THREAD

**Issue by koukichi7010** [skyline-emu/skyline-games-list#329]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Tested Build

#1099

### Android Version

11

### SoC type

Snapdragon

### Device GSMArena link

https://m.gsmarena.com/realme_6_pro-ampp-10114.php

### Game Behaviour

World End Syndrome boots and can be played but some things are broken or not showing up (e.g gameplay video)

It boots through intro and main menu

Gameplay can be accesed too but the background is not showing up
![Screenshot_2022-05-12-11-10-17-29_fcf072b9e4efc4ce5ca3e36691a7c3c6](https://user-images.githubusercontent.com/86339491/167986012-4a63bea8-332e-4e6e-ac56-9fa3eacb9a26.jpg)
Or you Can [Click Me](https://discord.com/channels/545842171459272705/960961451340754954/974147839426961469) to watch a screen record on skyline discord server as i can't seem to upload a video in here

### Is Log Level is set to 'Info'?

- [X] Yes

### Log file

[World End Syndrome.txt](https://github.com/skyline-emu/skyline-games-list/files/8674809/World.End.Syndrome.txt)

```
W1220HOS-0HandleRequest: Cannot find function in service 'visrv::IApplicationDisplayService': 0x836 (2102)

W1229GPFIFOSetProvokingVertex: Cannot set provoking vertex to last without host GPU support

**Comment by koukichi7010** [comment]
Alert #1107 fix 
Now the background is now fixed
![Screenshot_2022-05-14-09-24-43-10_fcf072b9e4efc4ce5ca3e36691a7c3c6](https://user-images.githubusercontent.com/86339491/168405758-f1906157-8257-4cf2-a2d7-6bee03b4c0f3.jpg)
![Screenshot_2022-05-14-09-24-23-63_fcf072b9e4efc4ce5ca3e36691a7c3c6](https://user-images.githubusercontent.com/86339491/168405764-8e9de583-4101-49fd-9afa-a1c3bf2e74f3.jpg)

**Comment by koukichi7010** [comment]
@K11MCH1 status ingame?!?!🤔🤔 Is it because the fps just goes down the further the game progresses?

**Comment by K11MCH1** [comment]
> @K11MCH1 status ingame?!?!🤔🤔 Is it because the fps just goes down the further the game progresses? Or fps degradation as you say it.

Yes. If the game has fps degradation, or the fps is less than half the game's fps cap then it is considered _in-game_.
Among other factors of course, but if one of those are present, then in-game.

Also, would you mind pinging me on Discord? Thanks!

**Comment by bylaws** [comment]
tagged playable since its a vn
